eth/core/precompile/method.go

```diff
@@ -31,6 +31,9 @@ import (
 	"pkg.berachain.dev/polaris/lib/utils"
 )
 
+// methodID is a fixed length byte array that represents the method ID of a precompile method.
+type methodID = [NumBytesMethodID]byte
```

Comment:
nit but I think this should be a type alias rather than a type declaration

Reply:
sure